src/Fixer/FunctionNotation/NoSpacesAfterFunctionNameFixer.php

```diff
@@ -65,6 +63,17 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
                 if (!$tokens[$possibleDefinitionIndex]->isGivenKind(T_FUNCTION)) {
                     $this->fixFunctionCall($tokens, $index);
                 }
+            } elseif ($tokens[$lastTokenIndex]->equalsAny($braceTypes)) {
+                $block = Tokens::detectBlockType($tokens[$lastTokenIndex]);
+                if (
```

Comment:
Altough this is not reported by fabbio.io or Travis, [PSR-2 does not allow a newline after the `(` or before the `)` of the `if` structure](http://www.php-fig.org/psr/psr-2/#overview):

> Opening parentheses for control structures MUST NOT have a space after them, and closing parentheses for control structures MUST NOT have a space before.

You should use `in_array()`, which would allow you to use such formatting in the array.

By the way, is there a fixer for this?